### PR TITLE
Update naming, margin, padding

### PR DESCRIPTION
**Pull request by DKnehi**

### CODE REVIEW

File: practice-two/src/styles/layout/footer.css [hunk 1]
```diff
@@ -11,18 +11,20 @@
...
 }
 
-.footer-socialbox {
+.footer-menu {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  margin-top: 25px;
+}
+.footer-social-box {
```
Comment (hoangle-agilityio) [inline practice-two/src/styles/layout/footer.css:27]: missing space empty

File: practice-two/src/styles/sections/process-management-section.css [hunk 2]
```diff
@@ -19,16 +19,18 @@
   font-size: var(--font-size-smallest);
   line-height: 22px;
   letter-spacing: 0.2px;
-  margin: 15px 0 40px 0;
+  margin-top: 15px;
+}
+.process-management-section-content-box {
```
Comment (hoangle-agilityio) [inline practice-two/src/styles/sections/process-management-section.css:24]: missing empty space

File: practice-two/src/styles/sections/quality-section.css [hunk 3]
```diff
@@ -29,10 +29,10 @@
   width: 100%;
 }
 
-.quality-section-second-imagebox {
+.quality-section-second-image-box {
   order: -1;
 }
-.quality-section-imagebox {
+.quality-section-image-box {
```
Comment (hoangle-agilityio) [inline practice-two/src/styles/sections/quality-section.css:35]: missing empty space. Verify all

File: practice-two/src/styles/layout/footer.css (hunk 1 above)
Comment (DKnehi) [reply practice-two/src/styles/layout/footer.css:27]: I will Fix

File: practice-two/src/styles/sections/process-management-section.css (hunk 2 above)
Comment (DKnehi) [reply practice-two/src/styles/sections/process-management-section.css:24]: i will fix

File: practice-two/src/styles/sections/quality-section.css (hunk 3 above)
Comment (DKnehi) [reply practice-two/src/styles/sections/quality-section.css:35]: i will fix

**Pull request merged**